program/src/stake_account.rs

```diff
@@ -121,6 +127,7 @@ pub fn deserialize_stake_account(account_data: &[u8]) -> Result<Stake, ProgramEr
 
     // We read 196 bytes from the data, so check that up front, so that bounds
     // checks can be optimized away below.
+    // I thought this was 200 the whole time, thanks for calculating this!
```

Comment:
I’m not sure if there is any data after it, it’s just that we only access the first 196 bytes.

Reply:
I think I went through the whole struct afterwards just to check if you were right, and I think it actually is `196`